### ISSUE TITLE
[feature request] Better Description & Expose the API search link, "Foogle"

### ISSUE THREAD

**Issue by kkweon** [Dobiasd/FunctionalPlus#82]
# Overview
**The Awesome repo needs an awesome description.**
So, I have two feature requests:

1. The current description is too implicative.
2. Add the Foogle to the description.

After all, it's just a suggestion, so feel free to close it if you think it's not necessary.

## 1. Too implicative description
By **Description**, I mean
![image](https://user-images.githubusercontent.com/2981167/29482116-dcf615e2-843f-11e7-84ba-839d2f8b64fb.png)
### Why?
- Sure it does help to write in concise and readable C++ code, but it does not truly show what `FunctionlPlus` is trying to offer. **Functional Programming Library for C++**
- When I search "Functional Programming C++," it doesn't show the `FunctionalPlus`.
 In order to search `FunctionalPlus`, you have to search "Concise C++ Code".
![image](https://user-images.githubusercontent.com/2981167/29482187-86d884c2-8441-11e7-93dd-098ce905decd.png)

### Possible suggestion
- Change to **Functional Programming Library for C++** or something similar

## 2. Expose Foogle
By **Foogle** I mean the API search website http://www.editgym.com/fplus-api-search/ 
It is truly awesome and super helpful, but it's currently hidden deeply in the README.

### Why?
In order to go to the Foogle from this GitHub,

you have to go through 3 steps.

1. Scroll down until "Finding the functions you need" in the toc
2. Click "Finding the functions you need"
3. Click "FunctionalPlus API search website."

### Possible suggestions 
- if you provide the link in the description, it's one click to the Foogle.
![image](https://user-images.githubusercontent.com/2981167/29482240-bf78e8ac-8442-11e7-9ada-2ac1711b8c6d.png)

- If you don't like it's in the description, I still think it should be located somewhere in the top of README.md, so it's easily accessible.

**Comment by Dobiasd** [comment]
Thanks. This is a very good observation, especially the problem with finding the library. I changed the  description like you suggested.

The name "foogle" is something I also initially thought about, since Haskel has its "Hoogle". However it sounded too much like "foo" to me and searching for this name on google already gives to many different results. And since I did not come up with a good unique name, I sticked with "fplus api search". :D
